app/views/facility_orders/index.html.haml

```diff
@@ -1,47 +1,53 @@
+= render 'shared/transactions/headers'
+
 = content_for :h1 do
   = current_facility
 
 = content_for :tabnav do
   = render partial: "admin/shared/tabnav_order", locals: { secondary_tab: "new"}
 
-- if @order_details.empty?
-  %p.notice There are no "New" or "In Process" orders.
-- else
-  = form_tag(batch_update_facility_orders_path, method: :post) do
-    - facility_users = User.find_users_by_facility(current_facility)
-    %table.table.table-striped.table-hover.old-table
-      = render partial: "table_controls"
-      %thead
-        -## THE HEADER ROW
-        %tr
-          %th
-          %th.order_num_width.centered= sortable "order_number", "Order"
-          %th.order_num_width.centered= "Order Detail"
-          %th.ordered_on_width= sortable "date", "Ordered On"
-          %th User Name
-          %th Quantity
-          %th.order-note= sortable "product"
-          %th= sortable "assigned_to"
-          %th= sortable "status"
-          %th.currency Price
+.span12
+  = render "shared/transactions/search2"
 
-      -## THE DETAIL ROWS
-      %tbody
-        - @order_details.each do |od|
+.span12
+  - if @order_details.empty?
```

Comment:
I would lean towards flipping the conditional so it's `if @order_details.any?` and the positive case is first.